### PR TITLE
LPS-166302 [Raylife AP] Create Quote Through Application Auto Flow

### PR DESCRIPTION
**Pull request by EugenioAntunes**
[LPS-166302](https://issues.liferay.com/browse/LPS-166302)

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 import {Parameters, parametersFormater} from '.';
 import {axios} from './liferay/api';
+const nowDate = new Date().toISOString().split('T')[0];
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts:17]: I believe we already have a current date variable on _dateFormatter_ folder. Try to find it.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts [hunk 2]
```diff
@@ -28,3 +29,18 @@ export function getQuotes(parameters: Parameters = {}) {
 
 	return axios.get(`${DeliveryAPI}/`);
 }
+
+const adaptQuoteRequest = (state: any) => ({
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts:37]: Try to type it correctly, if it's possible.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts [hunk 3]
```diff
@@ -28,3 +29,18 @@ export function getQuotes(parameters: Parameters = {}) {
...
+	billingOption: {
+		key: 'payInFull',
+		name: 'Pay in Full',
+	},
+	quoteCreateDate: nowDate,
+	r_applicationToQuotes_c_raylifeApplicationId: state?.applicationId,
+});
+
+export function createRaylifeAutoQuote(state: any) {
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts:46]: Same here.

**Review by joanagrego — CHANGES_REQUESTED** [review]
Nice job! I added some suggestions :call_me_hand:

**Comment by EugenioAntunes** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts (hunk 1 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts:17]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts (hunk 2 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts (hunk 3 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts]: Done

**Review by joanagrego — APPROVED** [review]
Approved! Thanks @EugenioAntunes :white_check_mark:

**Review by eliasantosf — APPROVED** [review]

**Comment by silasbsi** [comment]
ci:forward

**Comment by EugenioAntunes** [comment]
ci:forward

**Pull request closed without merging**